.ko.yaml

```diff
@@ -14,35 +14,35 @@
 # limitations under the License.
 
 # We need a shell for a lot of redirection/piping to work
-defaultBaseImage: ghcr.io/distroless/static:latest
```

Comment:
I recognize that this has moved, but to be honest, I'd prefer that that name of the base image be vendor neutral. Are there any distroless base images that are vendor neutral? I'm not very knowledgable in this space, so whatever you think is best.

Reply:
we were using that before, it just got renamed, but if you dont want we can swtich to the google distroless one, but imho i think this is more up to date :)

Reply:
I'm fine to use this! Just wanted to mention this since it might come up again.